static/home/js/matrix_utilities.js

```diff
@@ -1,3 +1,5 @@
+/*global Papa,d3,yadcf,showPDBtable,showAlert, _, signprotmat, con_seq, non_interactions,interactions_metadata,interactions,gprot,receptor*/
```

Comment:
what is this for?
helps codacy somehow?

Reply:
Jeps - we might want to exclude this, but it's good way to find errors in calls/variables/functions